Gems/ROS2/Code/Source/Lidar/LidarCore.cpp

```diff
@@ -55,6 +55,8 @@ namespace ROS2
 
     void LidarCore::ConfigureLidarRaycaster()
     {
+        m_lidarConfiguration.FetchLidarImplementationFeatures();
```

Comment:
@alek-kam-robotec-ai does the singleton called (eventually) in this line exist outside of the Editor context?